documentation/modules/security/con-securing-client-authorization.adoc

```diff
@@ -5,8 +5,10 @@
 [id='con-securing-client-authorization-{context}']
 = User authorization
 
-User authorization is configured using the `authorization` property in `KafkaUser.spec`.
-The authorization type enabled for a user is specified using the `type` field.
+[role="_abstract"]
+Use the `KafkaUser` custom resource to configure an authorization mechanism for clients that require access to a Kafka cluster. 
+Configure user authorization using the `authorization` property in `KafkaUser.spec`.
+Enable an authorization mechanism using the `type` field.
```

Comment:
Again, the authorization mechnaism is enabled and configured in the `KAfka` CR. in `KafkaUSer` you just configure the rules for given user.